app.js

```diff
@@ -22,9 +24,13 @@ app.get("/api/reviews/:review_id/comments", getReviewComments);
 
 app.get("/api/reviews", getAllReviews);
 
+app.post("/api/reviews/:review_id/comments", postComment);
+
 app.all("/*", notFound);
+
 app.use(badDataType);
 app.use(customErrors);
+app.use(lastResort);
```

Comment:
I like the order here and that "last resort" is exactly what it is! I would possibly recommend a renaming though - sound like variables rather than functions